spec/serializers/v3/product_serializer_spec.rb

```diff
@@ -76,4 +76,17 @@
       expect(nested_extension).to be_nil
     end
   end
+
+  describe 'Beta display' do
+    let(:products) { JSON.parse(file_fixture('products/dummy_products.json').read, symbolize_names: true) }
```

Comment:
When writting rspec (or any tests) it is good practice to make the individual specs (`it ...`) self-contained. So that you don't except other parts of your test framework to provide a certain state of test data.
I know it's not easy to get this right from the start (and I get it wrong still often enough), but this is a good bad example :wink: Because while this might work (Haven't tested it actually yet.), it's not very reliable. If someone would remove your Beta product from the fixture, your test wouldn't take notice, but become useless, since it couldn't even fail anymore, even if the serializer itself would show the release_stage again.

So instead of depending on the fixture, why not create your own product, only for this one spec, with exactly the data you need for the test. You can find other examples in this very file. Basically you just tell FactoryBot to `create(:product, ..., release_stage: 'boo!'`) and test the serialized output to not include the "boo".

That should be it :smiley_cat: